Makefile.fedora

```diff
@@ -0,0 +1,19 @@
+include Makefile
```

Comment:
I'm only 50/50 on adding a file like this as it is redundant with the defaults in Makefile. If we change something we now have to change it in two places. That said, it has the benefit of being consistent with the recent changes to add opensuse support.